app/src/sharedTest/java/org/oppia/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -0,0 +1,486 @@
+      onView(
+        allOf(
+          withId(R.id.slide_title_text_view),
+          isCompletelyDisplayed()
+        )
+      ).check(matches(withText(R.string.slide_3_title)))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_checkSlide3Description_isCorrect() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      SystemClock.sleep(500)
+      onView(
+        allOf(
+          withId(R.id.slide_description_text_view),
+          isCompletelyDisplayed()
+        )
+      ).check(matches(withText(R.string.slide_3_description)))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_checkSlide3Image_isCorrect() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      SystemClock.sleep(500)
+      onView(
+        allOf(
+          withId(R.id.slide_image_view),
+          isCompletelyDisplayed()
+        )
+      ).check(matches(withTagValue(equalTo(R.drawable.ic_onboarding_3))))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_checkSlide3_skipButtonIsNotVisible() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      SystemClock.sleep(500)
+      onView(withId(R.id.skip_text_view)).check(matches(not(isDisplayed())))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_checkSlide3_getStartedButtonIsVisible() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      SystemClock.sleep(500)
+      onView(withId(R.id.get_started_button)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_checkSlide3_clickGetStartedButton_opensProfileActivity() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      onView(withId(R.id.onboarding_slide_view_pager)).perform(swipeLeft())
+      SystemClock.sleep(500)
+      onView(withId(R.id.get_started_button)).perform(click())
+      intended(hasComponent(ProfileActivity::class.java.name))
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_swipeLeftOnLastSlide_isNotWorking() {
```

Comment:
Here and for similarly named tests, I think the outcome should be something like ‘doesNotWork’ rather than is not working. I think that might read a bit better.

Reply:
Updated as per comment.